internal/dao/dao_errors.go

```diff
@@ -8,8 +8,17 @@ import (
 
 	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
 	"github.com/go-playground/validator/v10"
+	"github.com/jackc/pgx/v5"
 )
 
+// TODO let's vastly simplify error handling in DAO, let's just create aliases for native PGX errors
+// and that should be enough:
+
+var ErrNoRows = pgx.ErrNoRows
+var ErrAffectedMismatch = errors.New("unexpected affected rows")
```

Comment:
I guess linter wants to group the var statements, right?

Reply:
Yeah.